### PR TITLE
fix eslint errors in components

### PR DESCRIPTION
**Pull request by pallabez**

### CODE REVIEW

File: components/PopupAssignee.vue [hunk 1]
```diff
@@ -55,13 +64,13 @@ export default {
   },
   watch: {
     localAssignees: {
-      handler(newValue) {
+      handler (newValue) {
```
Comment (shubhamsigdar1) [inline components/PopupAssignee.vue:67]:
```suggestion
      handler(newValue) {
```

File: components/PopupAssignee.vue [hunk 2]
```diff
@@ -55,13 +64,13 @@ export default {
...
         this.$emit('input', newValue)
       },
       deep: true
     }
   },
-  mounted() {
+  mounted () {
```
Comment (shubhamsigdar1) [inline components/PopupAssignee.vue:73]:
```suggestion
  mounted() {
```

File: components/PopupAssignee.vue [hunk 3]
```diff
@@ -33,14 +42,14 @@
...
-  data() {
+  data () {
     return {
       localAssignees: this.assignee,
       name: 'Assignees'
     }
   },
   computed: {
-    users() {
+    users () {
```
Comment (shubhamsigdar1) [inline components/PopupAssignee.vue:52]:
```suggestion
    users() {
```

File: components/PopupAssignee.vue [hunk 4]
```diff
@@ -33,14 +42,14 @@
 export default {
   name: 'PopupAssignee',
   props: ['assignee'],
-  data() {
+  data () {
```
Comment (shubhamsigdar1) [inline components/PopupAssignee.vue:45]:
```suggestion
  data() {
```

File: components/PopupAssignee.vue (hunk 1 above)
Comment (shubhamsigdar1) [reply components/PopupAssignee.vue:67]: Can you check this? I don't know is it right or wrong but I think there is no space between function name and bracket

Comment (pallabez) [reply components/PopupAssignee.vue:67]: This is opinionated from code base to code base. The Eslint is configured that way.

**Review by shubhamsigdar1 — APPROVED** [review]
LGTM

**Pull request merged**